js/data.js

```diff
@@ -1 +1,15 @@
 /* exported data */
+var data = {
+  entries: [],
+  view: ''
+};
+
+window.addEventListener('beforeunload', function (event) {
+  var formJSON = JSON.stringify(data);
+  localStorage.setItem('javascript-local-storage', formJSON);
```

Comment:
Probably update this name from `'javascript-local-storage'` to `'party-local-storage'`